lib/views/help/credits.html.erb

```diff
@@ -1,91 +1,45 @@
-<% @title = "Credit where credit is due" %>
+<% @title = "Agradecimientos" %>
 
 <%= render :partial => 'sidebar' %>
 <div id="left_column_flip">
 
-<h1 id="credits"><%= @title%> <a href="#credits">#</a> </h1>
+<h1 id="credits"><%= @title%> <a href="#agradecimientos">#</a> </h1>
```

Comment:
I don't know whether this anchor is actually linked to from elsewhere, but typically we keep them as english so that any existing links are not broken. I'll revert this line on merge. Cheers!